pkg/kritis/crd/securitypolicy/securitypolicy.go

```diff
@@ -51,9 +52,26 @@ func ImageSecurityPolicies(namespace string) ([]v1beta1.ImageSecurityPolicy, err
 	return list.Items, nil
 }
 
+type Validator struct {
+	imageToVulnz map[string][]metadata.Vulnerability
```

Comment:
Since caching inevitably causes confusion for new developers, do you mind adding a comment that specifically calls out that this is used as a cache?

Reply:
sure! done